lib/appliance_console/external_httpd_configuration.rb

```diff
@@ -18,8 +18,9 @@ module ExternalHttpdConfiguration
       HTTP_KEYTAB         = "/etc/http.keytab"
 
       EXTERNAL_AUTH_FILE  = "conf.d/cfme-external-auth"
-      HTTPD_EXTERNAL_AUTH = "/etc/httpd/#{EXTERNAL_AUTH_FILE}"
-      HTTPD_CONFIG        = "/etc/httpd/conf.d/cfme-https-application.conf"
+
+      HTTPD_EXTERNAL_AUTH = Pathname.new(ENV['MIQ_APACHE_ROOT_DIR']).join("etc/httpd/#{EXTERNAL_AUTH_FILE}").to_s
+      HTTPD_CONFIG        = Pathname.new(ENV['MIQ_APACHE_ROOT_DIR']).join("etc/httpd/conf.d/cfme-https-application.conf").to_s
```

Comment:
Again, not sure what's using these values but most things should handle Pathname objects just fine.